app/src/main/res/drawable/ic_notifications_anim_disable.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<animated-vector
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:drawable="@drawable/ic_notifications_enabled">
-    <target android:name="strike_thru_path" android:animation="@animator/strike_thru_path_disable" />
```

Comment:
Can we also remove `strike_thru_path_disable.xml`

Reply:
Removed. Thanks!